provision/provision-site.sh

```diff
@@ -23,13 +23,38 @@ NGINX_UPSTREAM=$6
 VVV_PATH_TO_SITE=${VM_DIR} # used in site templates
 VVV_SITE_NAME=${SITE}
 VVV_HOSTS=""
-
 SUCCESS=1
 
 VVV_CONFIG=/vagrant/config.yml
 
 . "/srv/provision/provisioners.sh"
 
+# @description Retrieves a config value for the given site as specified in `config.yml`
+#
+# @arg $1 string the config value to fetch
+# @arg $2 string the default value
+function vvv_get_site_config_value() {
+  local value=$(shyaml -q get-value "sites.${SITE_ESCAPED}.${1}" "${2}" < ${VVV_CONFIG})
+  echo "${value}"
+}
+
+# @description Reset the PHP global version to the default
```

Comment:
not quite true as it checks the site config value, so if the default is 7.4 but my site says `php: 8.0` then this function sets PHP to v8, not v7.4

Reply:
```suggestion
# @description sets a sites PHP version as the global version, or to the VVV default if none is specified
```